src/components/fabSpeedDial/demoModal/style.scss

```diff
@@ -0,0 +1,16 @@
+.md-fab.md-mini {
+  background-color: #aaa !important;
```

Comment:
I've replaced it with the following (will commit soon):

```scss
.md-fab.demo-fab.trigger-fab, .md-fab.demo-fab.action-fab {
  &:hover, &.md-focused {
    background-color: #333;
  }
}

.md-fab.demo-fab.action-fab {
  background-color: #aaa;
}
```

In order to make this work, I wound up having to add two new classes (`.demo-fab` and `.trigger/action-fab`) to each one in order to override the extreme specificity of the default fab. Is there a better way to do this?